Clinical-Genomics/scout
# Load all pathogenic

This PR changes so that scout will load all variants with a clinvar annotation in `['pathogenic', 'likely_pathogenic', 'conflicting_interpretations_of_pathogenecity']`

**How to test**:
There are some new test files to check that the behaviour works:

There are two small VCFs in `tests/fixtures/vcfs/` with two variants in them. One file (`no_clinvar.vcf`) lacks clinvar annotation. Only one of the variants should be loaded (because of rank score). The other file (`clinvar.vcf`) have the same variants but with clinvar annotation. When this file is used, both variants should be loaded.

1. Install this branch
1. Load the case without clinvar and check that only one variant is loaded.
```scout load case --vcf tests/fixtures/vcfs/no_clinvar.vcf --ped tests/fixtures/peds/643594.ped --owner cust000``` 
Check that only one variant is loaded
1. Delete the case: `scout delete case --case-id clinvar`
1. Load case again with the file that have clinvar annotation
```scout load case --vcf tests/fixtures/vcfs/clinvar.vcf --ped tests/fixtures/peds/643594.ped --owner cust000``` 
1. Check that both variants are loaded regardless of rank score.

**Expected outcome**:
All variants with the given clinvar annotations should be loaded

**Review:**
